x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -189,35 +206,38 @@ export const HostList = () => {
       },
       {
         field: 'metadata.agent.version',
-        name: i18n.translate('xpack.securitySolution.endpoint.host.list.endpointVersion', {
+        name: i18n.translate('xpack.securitySolution.endpointList.endpointVersion', {
           defaultMessage: 'Version',
         }),
       },
       {
-        field: '',
-        name: i18n.translate('xpack.securitySolution.endpoint.host.list.lastActive', {
-          defaultMessage: 'Last Active',
-        }),
-        dataType: 'date',
-        render: () => {
-          return 'xxxx';
+        field: 'metadata.@timestamp',
+        name: lastActiveColumnName,
+        render(dateValue: HostInfo['metadata']['@timestamp']) {
+          return (
+            <FormattedDate
+              fieldName={lastActiveColumnName}
+              value={dateValue}
+              className="eui-textTruncate"
+            />
+          );
         },
       },
     ];
   }, [queryParams]);
 
   return (
-    <PageView
+    <ManagementPageView
       viewType="list"
       data-test-subj="hostPage"
-      headerLeft={i18n.translate('xpack.securitySolution.endpoint.host.hosts', {
-        defaultMessage: 'Hosts',
+      headerLeft={i18n.translate('xpack.securitySolution.endpointLis.pageTitle', {
```

Comment:
typo -> `endpointList`

Reply:
ahhhh. 😐 
I will update it if the build fails. If it does not fail, I will make the change in my next PR. Thanks